icarus_backend/drone/DroneViews.py

```diff
@@ -47,3 +47,25 @@ def register_drone(request):
     response_data = {'message': 'Successfully registered this drone.'}
     response_json = json.dumps(response_data)
     return HttpResponse(response_json, content_type="application/json")
+
+@protected_resource()
+@api_view(['POST'])
+def edit_drone_details(request):
+    body = request.data
+    drone_id = [d['id'] for d in body]
+    new_drone = Drone(id=drone_id, owner=request.user, description=body['description'],
+                      manufacturer=body['manufacturer'], type=body['type'],
+                      color=body['color'])
+    drone = Drone.objects.filter(id__in=drone_id).first()
```

Comment:
yah again I think if you just did body['drone_id'] you would be able to do `Drone.objects.filter(id=drone_id).first()` and it is functionally the same but maybe a little more clear.